tensorflow_app/views/import_graphdef.py

```diff
@@ -55,21 +55,34 @@ def get_padding(node, layer):
     pad_top = pad_along_height / 2
     pad_left = pad_along_width / 2
     '''
-
-    pad_h = ((int(output_shape[1]) - 1) * layer['params']['stride_h'] +
-             layer['params']['kernel_h'] - int(input_shape[1])) / float(2)
-    pad_w = ((int(output_shape[2]) - 1) * layer['params']['stride_w'] +
-             layer['params']['kernel_w'] - int(input_shape[2])) / float(2)
-
-    # check this logic (see caffe-tensorflow/kaffe/shapes.py)
-    if node.type == "Conv2D":
+    if node.type == "Conv3D":
+        pad_d = ((int(output_shape[1]) - 1) * layer['params']['stride_h'] +
```

Comment:
I think in the params should have been `stride_d` and `kernel_h`.

Reply:
Oh sorry my bad